modules/aws/ssm.go

```diff
@@ -0,0 +1,72 @@
+package aws
+
+import (
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ssm"
+)
+
+func GetParameter(t *testing.T, awsRegion string, keyName string) string {
+	keyValue, err := GetParameterE(t, awsRegion, keyName)
+	if err != nil {
+		t.Fatal(err)
+	}
+	return keyValue
+}
+
+func GetParameterE(t *testing.T, awsRegion string, keyName string) (string, error) {
+	ssmClient, err := NewSsmClientE(t, awsRegion)
+	if err != nil {
+		return "", err
+	}
+
+	resp, err := ssmClient.GetParameter(&ssm.GetParameterInput{Name: aws.String(keyName), WithDecryption: aws.Bool(true)})
+	if err != nil {
+		return "", err
+	}
+
+	parameter := *resp.Parameter
+	return *parameter.Value, nil
```

Comment:
Should this use `aws.StringValue`?

Reply:
And do you need a `nil` check for the previous line?

Reply:
Empty value versions are not allowed in SSM Parameter Store (https://forums.aws.amazon.com/message.jspa?messageID=849170) and you would instead get an error 400 is a version does not exist. I don't think it's necessary to check for `nul` values here in this case.